shared/config/defaults.go

```diff
@@ -0,0 +1,7 @@
+package config
+
+const (
```

Comment:
NOTE: Will need to be refactored after merging with https://github.com/pokt-network/pocket/pull/166